apps/api/v2/src/ee/bookings/2024-08-13/controllers/e2e/user-bookings.e2e-spec.ts

```diff
@@ -1936,292 +1938,465 @@ describe("Bookings Endpoints 2024-08-13", () => {
+              language: "it",
+            },
+            location: {
+              type: "integration",
+              integration: "google-meet",
+            },
+          };
 
-        const bookingResponse = await request(app.getHttpServer())
-          .post("/v2/bookings")
-          .send(bookingBody)
-          .set(CAL_API_VERSION_HEADER, VERSION_2024_08_13)
-          .expect(201);
+          await request(app.getHttpServer())
+            .post("/v2/bookings")
+            .send(bookingBody)
+            .set(CAL_API_VERSION_HEADER, VERSION_2024_08_13)
+            .expect(400);
+        });
 
-        const bookingResponseBody: CreateBookingOutput_2024_08_13 = bookingResponse.body;
-        const createdBooking = bookingResponseBody.data;
-        expect(createdBooking).toHaveProperty("id");
+        it("can't book with invalid location type", async () => {
+          const bookingBody = {
+            start: new Date(Date.UTC(2040, 0, 9, 13, 0, 0)).toISOString(),
+            eventTypeId: eventTypeWithAllLocationsId,
+            attendee: {
+              name: "Mr Proper",
+              email: "mr_proper@gmail.com",
+              timeZone: "Europe/Rome",
+              language: "it",
+            },
+            location: {
+              type: "blablabala",
+            },
+          };
 
-        if (responseDataIsBooking(createdBooking)) {
-          expect(createdBooking.location).toEqual(attendeeAddress);
-          await bookingsRepositoryFixture.deleteById(createdBooking.id);
-        } else {
-          throw new Error("Unexpected response data type");
-        }
+          await request(app.getHttpServer())
+            .post("/v2/bookings")
+            .send(bookingBody)
+            .set(CAL_API_VERSION_HEADER, VERSION_2024_08_13)
+            .expect(400);
+        });
       });
 
-      xit("can book with attendeeAddress location", async () => {
-        const attendeePhone = "+37120993151";
+      describe("Can update booking with different locations", async () => {
+        const email = `user-booking-locations-${randomString(10)}@gmail.com`;
+        const username = `user-bookings-locations-${randomString(10)}`;
+        const locale = "en";
+        const timeZone = "Europe/London";
+        const attendee = await userRepositoryFixture.create({
+          email,
+          username,
+          locale,
+          timeZone,
+        });
 
-        const bookingBody: CreateBookingInput_2024_08_13 = {
-          start: new Date(Date.UTC(2040, 0, 9, 13, 0, 0)).toISOString(),
-          eventTypeId: eventTypeWithAllLocationsId,
-          attendee: {
-            name: "Mr Proper",
-            email: "mr_proper@gmail.com",
-            timeZone: "Europe/Rome",
-            language: "it",
+        const booking = await bookingsRepositoryFixture.create({
+          uid: `booking-uid-${randomString(10)}`,
+          title: "booking title",
+          startTime: "2060-09-05T11:00:00.000Z",
```

Comment:
Did the tests pass for you locally? I am asking because this booking has startTime `"2060-09-05T11:00:00.000Z"` and we use the same startTime for test `it("should reschedule", async () => {` - I think it would result in a conflict??